tests/Functional/Fixtures/web/symfony.php

```diff
@@ -21,4 +21,4 @@
 $request = Request::createFromGlobals();
 $response = $kernel->handle($request);
 $response->send();
-$kernel->terminate($request, $response);
+//$kernel->terminate($request, $response);
```

Comment:
oh hell, i dug some more. the dispatcher is always null. in symfony, you create a Kernel which gets HttpKernel from the container. the event dispatcher is created and injected by the container... i need to fix the setup of the kernel then.